handlers/oam_handler.go

```diff
@@ -95,7 +95,8 @@ func (h *Handler) PatternFileHandler(
 	if h.kubeclient.DynamicKubeClient == nil {
 		kc, err := meshkube.New(prefObj.K8SConfig.Config)
 		if err != nil {
-			logrus.Error("failed to create kube client: ", err)
+			h.log.Error(ErrKubeClient(err))
+			http.Error(w, ErrKubeClient(err).Error(), http.StatusInternalServerError)
```

Comment:
In line number 47 the `http.ResponseWriter` is rw so here consider writing this statement as `http.Error(rw, ErrKubeClient(err).Error(), http.StatusInternalServerError)`